app/mailers/app_mailer.rb

```diff
@@ -8,4 +8,9 @@ def password_reset(user)
     @user = user
     mail from: 'no-reply@myflix.com', to: user.email_address, subject: 'Reset Password - MyFliX'
   end
+
+  def invitation(invitation)
+    @invitation = invitation
+    mail from: 'no-reply@myflix.com', to: invitation.email, subject: 'Myflix Invite'
```

Comment:
You could refactor that 'from:' option to above and use something like `default from: 'address'` - works same way as before_action in controller.